### PR TITLE
Feature/driver public profile

### PR DESCRIPTION
**Pull request by kaharlykskyi**

### CODE REVIEW

File: resources/assets/js/features/public-profiles/components/DriverBasicInfo.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import "../styles/driver-profile.scss";
+
+class DriverBasicInfo extends React.Component {
+
+    render() {
+        const { first_name, last_name, years, img, translate } = this.props;
+
+        return (
+            <div className="text-center">
+                <img src={img ? (img) : 'http://www.theenergycollective.com/wp-content/plugins/userpro/img/default_avatar_male.jpg'} width="150" height="150" className="rounded-circle driver-avatar" />
```
Comment (TatsuUkraine) [inline resources/assets/js/features/public-profiles/components/DriverBasicInfo.js:13]: create PhotoService or something like that and move default img path definition to const

File: resources/assets/js/features/public-profiles/components/DriverProfileContainer.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        super(props);
+
+        this.state = {
+            profile: {},
+            preloader: true,
+        }
+    }
+
+    componentDidMount() {
+        const profile = DriverProfileService.getDriverProfile(this.props.id);
```
Comment (TatsuUkraine) [inline resources/assets/js/features/public-profiles/components/DriverProfileContainer.js:21]: profile should be from global state

Couple of notes:
1. you can create global state for profile collection and search by ID there. In addition - use redux suggestions for data normalization, you can store profile collection like described here http://redux.js.org/docs/recipes/reducers/NormalizingStateShape.html and keep comments as key in profile or you can move comments to separate property
2. you can create state for single profile that will be viewed on this page

any variant can be used here, you just need to choose right one depends on next criteria: will you have list of profile, will it be only ne profile always and etc.

File: resources/assets/js/features/public-profiles/components/DriverRating.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                        <div className="rating-block">
+                            <h5>{translate('driver_public_profile.driver_rating')}</h5>
+                            <h2 className="bold padding-bottom-7"><i className="fa fa-star" aria-hidden="true"></i>4.3 <span className="small-text">/ 5</span></h2>
+                        </div>
+                    </div>
+                    <div className="col-sm-6">
+                        <h5>{translate('driver_public_profile.driver_rating_breakdown')}</h5>
+                        <div className="pull-left">
+                            <div className="pull-left" style={{width: '35px', lineHeight: 1}}>
+                                <div style={{height:'9px', margin: '5px 0'}}>5</div>
```
Comment (TatsuUkraine) [inline resources/assets/js/features/public-profiles/components/DriverRating.js:24]: avoid inline styles

**Review by sergey-ben — APPROVED** [review]

File: resources/assets/js/features/public-profiles/services/DriverProfileService.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                    }
+                ]
+            };
+        /*return securedRequest.get('/api/v1/driver/' + id)
+            .then(
+                response => Promise.resolve(response.data),
+                error => Promise.reject(error.response.data)
+            );*/
+    },
+    getTimeFromDate(date, get) {
```
Comment (TatsuUkraine) [inline resources/assets/js/features/public-profiles/services/DriverProfileService.js:43]: move this method to DateTime helpert

**Pull request merged**